src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -0,0 +1,237 @@
+        chartLabel.setText(CHART_LABEL);
+        availableToday.setText(AVAILABLE_LABEL);
+        setUpList();
+        setUpAnimation();
+    }
+
+    /**
+     * Sets up listView elements.
+     */
+    private void setUpList() {
+        availableList.setItems(getAvailableTodayList());
+        availableList.setCellFactory(listView -> new AvailableTodayCell());
+        availableList.setFocusTraversable(false);
+        availableList.setMouseTransparent(true);
+    }
+
+    private void setUpAnimation() {
+        moveTransition = new TranslateTransition(Duration.millis(MOVE_DURATION), this.getRoot());
+        double originalPosition = this.getRoot().getTranslateX();
+        moveTransition.setFromX(originalPosition + MOVE_INITIAL);
+        moveTransition.setToX(originalPosition - MOVE_POP);
+        moveBackTransition = new TranslateTransition(Duration.millis(MOVE_DURATION), this.getRoot());
+        moveBackTransition.setFromX(originalPosition - MOVE_POP);
+        moveBackTransition.setToX(originalPosition);
+        moveBackTransition.setDelay(Duration.millis(MOVE_DURATION));
+        fadeInTransition = new FadeTransition(Duration.millis(FADE_DURATION), this.getRoot());
+        fadeInTransition.setFromValue(FADE_INITIAL);
+        fadeInTransition.setToValue(FADE_ULTIMATE);
+        fadeInTransition.playFromStart();
+        moveTransition.playFromStart();
+        moveBackTransition.playFromStart();
+    }
+
+
+    /**
+     * @return List of people who are available today.
+     */
+    public ObservableList<Person> getAvailableTodayList() {
+
+        Day filterDay = Day.MONDAY;
+        DateTimeFormatter dayFormatter = DateTimeFormatter.ofPattern("EEEE");
+        String today = LocalDateTime.now().format(dayFormatter);
+        switch (today) {
+        case "Monday":
+            filterDay = Day.MONDAY;
+            break;
+        case "Tuesday":
+            filterDay = Day.TUESDAY;
+            break;
+        case "Wednesday":
+            filterDay = Day.WEDNESDAY;
+            break;
+        case "Thursday":
+            filterDay = Day.THURSDAY;
+            break;
+        case "Friday":
+            filterDay = Day.FRIDAY;
+            break;
+        case "Saturday":
+            filterDay = Day.SATURDAY;
+            break;
+        case "Sunday":
+            filterDay = Day.SUNDAY;
+            break;
+        default:
+            break;
+        }
+        final Day effectiveFilterDay = filterDay;
+        return personList.filtered(person -> person.getDaysAvailable().contains(effectiveFilterDay));
+    }
+
+    /**
+     * @return Get the amount of contacts in FriendFolio
+     */
+    public Integer getContactAmount() {
+        return personList.size();
+    }
+
+    /**
+     * @return Get the amount of debt the user has
+     */
+    public double getTotalDebt() {
+
+        double result = 0.0;
+        for (Person x : personList) {
+            if (x.getMoneyOwed().getAmount() > 0) {
```

Comment:
In MoneyOwed class there is a instance level function called `userOwesMoney`. You can use that instead. Also if I'm not wrong, negative means user owes money, so you can just absolute the value.

This and the above comment also applies to your credit function.